src/status_im/ui/screens/wallet/main/views.cljs

```diff
@@ -68,7 +68,7 @@
   [{:label               (i18n/label :t/wallet-send)
     :accessibility-label :send-transaction-button
     :icon                :main-icons/send
-    :action              #(re-frame/dispatch [:navigate-to :wallet-send-transaction])}
+    :action              #(re-frame/dispatch [:navigate-to :wallet-send-transaction-new])}
```

Comment:
can you namespace new screens?

Reply:
this name is temporary, it will be back to normal `:wallet-send-transaction` after dapp and chat transactions are also merged and old wallet flow is removed. 
Or did you mean, namespace them, in general, e.g. `:wallet/send-transaction..` etc?